style2.css

```diff
@@ -0,0 +1,294 @@
+@media screen and (max-width: 1440px) {
+	
+	
+}
+
+@media screen and (max-width: 1120px) {
+	.header__menu-link {
+		font-size: 14px;
+	}
+}
+@media screen and (max-width: 1090px) {
+.header__menu-link {
+	font-size: 13px;
+}
+}
+
+@media screen  and (max-width: 1062px) {
+.menu{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)